Makefile

```
@@ -237,88 +258,4 @@ docker-restart-8001: ## run -d --name bmi-app -p 8001:8000 bmi-app:dev
-	@echo "[bandit] full repo scan with safe excludes"
-	bandit -q -r . \
-	  -x frontend,node_modules,dist,build,test-results,.venv,venv,cache,tests \
-	  -s B101 || true
-
-lint:
-	ruff check .
-	black --check .
-	mypy app scripts
-
-.PHONY: help venv dev test cov cov-html lint fmt smoke-auto smoke-8000 smoke-8001 docker-build docker-run docker-run-bg docker-stop docker-restart-8001 bandit bandit-full
+.PHONY: help venv setup-automation dev test test-fast cov cov-check cov-html lint fmt fmt-check security pre-commit quick-check auto-push safe-push feature sync-main status clean clean-coverage clean-backups clean-tests check-all fix-all ci smoke-auto smoke-8000 smoke-8001 docker-build docker-run docker-run-bg docker-stop docker-restart-8001 bandit bandit-full
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)